packages/app/src/lib/components/TimelineView.svelte

```diff
@@ -167,35 +166,12 @@
     replyingTo = undefined;
   }
 
-  //
-  // Settings Dialog
-  //
-
-  $effect(() => {
-    if (!globalState.space) return;
-
-    globalState.space &&
-      globalState.space.sidebarItems.items().then((items) => {
-        for (const item of items) {
-          const category = item.tryCast(Category);
-          if (
-            category &&
-            typeof category.channels === "object" &&
-            typeof category.channels.ids === "function" &&
-            globalState.channel &&
-            category.channels.ids().includes(globalState.channel.id)
-          ) {
-            return;
-          }
-        }
-      });
-  });
```

Comment:
@codegod100 mind verifying that this code is no longer needed? I think you were the last person to touch it. Perhaps it was a merge conflict? Just doesn't seem to actually do anything.

Reply:
Yeah this seems like abandoned code I forgot to remove when I was implementing images

Reply:
So you're telling me it was MY FAULT!?! haha looks like it was. When I moved the Dialog I forgot to remove this. Cool. Thanks for calling that out.